libs/tree-sitter-wing/grammar.js

```diff
@@ -332,10 +332,13 @@ module.exports = grammar({
 
     bool: ($) => choice("true", "false"),
 
-    duration: ($) => choice($.seconds, $.minutes, $.hours),
+    duration: ($) => choice($.seconds, $.minutes, $.hours, $.days, $.months, $.years),
     seconds: ($) => seq(field("value", $.number), "s"),
     minutes: ($) => seq(field("value", $.number), "m"),
     hours: ($) => seq(field("value", $.number), "h"),
+    days: ($) => seq(field("value", $.number), "d"),
+    months: ($) => seq(field("value", $.number), "mo"),
+    years: ($) => seq(field("value", $.number), "y"),
```

Comment:
I dont think we currently have duration expressions in our tree-sitter tests but maybe we should add them